source/manual/github-unavailable.html.md

```diff
@@ -1,68 +1,21 @@
 ---
 owner_slack: "#govuk-developers"
-title: Deploy when GitHub is unavailable
+title: When GitHub is unavailable
 section: Deployment
 layout: manual_layout
 parent: "/manual.html"
 ---
 
-Many of the git repositories which make up GOV.UK are hosted on public GitHub. If GitHub becomes unavailable, our Jenkins deployment pipeline breaks. This is because:
+## Backup to AWS CodeCommit
 
-* Jenkins uses the [GitHub Authentication plugin](https://plugins.jenkins.io/github-oauth/) to manage access: [only people in certain GitHub groups can access Jenkins](/manual/access-jenkins.html). If GitHub goes down, nobody can log into Jenkins
-* The [`Deploy_App` Jenkins job][Deploy_App]  uses GitHub by default when it clones repositories ready for deploying. If GitHub goes down, it can't clone the repository
+We mirror all non-archived GitHub repositories tagged with `govuk` to AWS CodeCommit. This is achieved using a Mirror GitHub repositories workflow, which is [configured in terraform](https://github.com/alphagov/govuk-infrastructure/blob/main/terraform/deployments/github/mirror.tf).
```

Comment:
We manage the code commit repositories (i.e. create/delete the repos) using Terraform. But the actual syncing the data between GitHub and those CodeCommit repos happens in a GitHub Action workflow here:https://github.com/alphagov/govuk-infrastructure/blob/main/.github/workflows/mirror-repos.yml

Reply:
Have updated to include reference to the Github action.